README.md

```diff
@@ -1,2 +1,233 @@
+![structure](https://user-images.githubusercontent.com/15249242/95333060-e34f3600-08c9-11eb-91e3-8ea64446eb77.PNG)
+
+### Trigger authentication
+In the `<APP_HOME>/LoginPage.xaml` page, we have registered a `Click` event named `LoginButton_Click` for the login button to trigger a OIDC authentication:
+```xml
+<Button x:Name ="login" Content="LOGIN" HorizontalAlignment="Left" Margin="73,452,0,0" VerticalAlignment="Top" 
+                Width="240" Height="56" FontFamily="Segoe UI Black" FontSize="22" Click="LoginButton_Click" Background="LightSeaGreen" Foreground="Black"/>
+```
+
+The button click would trigger an authentication request, and redirect the user to the IdP authentication page.
+Upon successful authentication, the user would be redirected to the application homepage.
+
+### Retrieve user attributes
+In the `<APP_HOME>/LoginPage.xaml.cs` file, we have added the following code inside the login button click trigger method to get the user subject value and the user attributes referring the SDK API.
+
+```csharp
+private async void LoginButton_Click(object sender, RoutedEventArgs e)
+{
+    // Redirect the user to IDP authentication page
+    await authenticationHelper.Login();
+
+    // Focus to app windows after succeful authentication
+    this.Activate();
+
+    // Retrieve access token and user information
+    accessToken = authenticationHelper.AccessToken;
+    userInfo = authenticationHelper.UserInfo;
+
+    // Display the home page window
+    HomePage home = new HomePage(accessToken, userInfo);
+    home.Show();
+    this.Close();
+}
+```
+
+### Trigger logout
+In the `<APP_HOME>/HomePage.xaml` file, we have added the following button to trigger a SLO flow:
+```xml
+<Button x:Name="logoutButton" Width="Auto" Foreground="White" Background="Black" FontWeight="Bold" 
+        FontSize="14" Click="Logout_button_click" Margin="944,10,9.6,650.2">
+    <StackPanel Orientation="Horizontal">
+        <Image Source="Assets/logout-white.png" Width="20" Height="20"/>
+    </StackPanel>
+</Button>
+```
+
+Clicking on the logout link would trigger the SLO flow.
+
+## Integrating OIDC SDK to your existing .NET application
+This section will guide you on integrating OIDC into your existing .NET application with the Asgardio Dotnet OIDC SDK.
+This allows a .NET application (i.e. Service Provider) to connect with an IDP using OpenID Connect protocol.
+This guide consist with the following sections.
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Installing the SDK](#installing-the-sdk)
+  * [Using Nuget Package Manager](#using-nuget-package-manager)
+  * [Using the library DLL](#using-the-library-dll)
+- [Login](#login)
+- [Logout](#logout)
+- [Get User Info](#get-user-info)
+
+### Introduction
+These instructions will guide you on integrating OIDC into your .NET application with the Asgardio Dotnet OIDC SDK. This allows an application (i.e. Service Provider) to connect with an IDP using OIDC protocol.
```

Comment:
This is mentioned in L130-131 as well. Do we need to repeat it here?

Reply:
Resolved in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/4/commits/3945c817476dfe36be1ba0eac68d777f7d6dd006